### PR TITLE
Added Geokeo geocoding API in alphabetical order

### PR DESCRIPTION
**Pull request by thefool76**
- [ ✔] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [✔ ] My addition is ordered alphabetically
- [✔ ] My submission has a useful description
- [ ✔] The description does not end with punctuation
- [✔ ] Each table column is padded with one space on either side
- [ ✔] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by thefool76** [comment]
Done!

**Review by Loyoki — APPROVED** [review]

**Comment by matheusfelipeog** [comment]
> hey @Loyoki when are u gonna merge this pr?

Only maintainers can do this. I'll review your PR now ;)

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
@thefool76 Can you resolve this review, please?

**Comment by matheusfelipeog** [comment]
refs:

- Issue #2002
- Original PR: #1916

File: README.md [hunk 1]
```diff
@@ -620,6 +620,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Geocoding.ai](https://nominatim.geocoding.ai/) | Provides worldwide forward / reverse geocoding | No | Yes | Yes |
 | [GeoDataSource](https://www.geodatasource.com/web-service) | Geocoding of city name by using latitude and longitude coordinates | `apiKey` | Yes | Unknown |
 | [GeoJS](https://geojs.io/) | IP geolocation with ChatOps integration | No | Yes | Yes |
+| [Geokeo](https://geokeo.com) | Geokeo geocoding service- with 2500 free api requests daily | No | No | Yes |
```
[the remark this answers is not in the document]
Comment (thefool76) [reply README.md:623]: @matheusfelipeog Yes sir of course!

**Comment by thefool76** [comment]
@matheusfelipeog sorry for the mistake

**Comment by matheusfelipeog** [comment]
> @matheusfelipeog sorry for the mistake

No problem ;)

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for new contribution @thefool76 ;)

**Pull request merged**